app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V1
+    class ApiController < ApplicationController
+      include Concerns::ActAsApiRequest
+      include DeviseTokenAuth::Concerns::SetUserByToken
+
+      before_action :authenticate_user!, except: :status
+
+      layout false
+      respond_to :json
+
+      rescue_from ActiveRecord::RecordNotFound,        with: :render_not_found
+      rescue_from ActiveRecord::RecordInvalid,         with: :render_record_invalid
+      rescue_from ActionController::RoutingError,      with: :render_not_found
+      rescue_from AbstractController::ActionNotFound,  with: :render_not_found
+
+      def status
+        render json: { online: true }
+      end
+
+      def render_not_found(exception)
+        logger.info(exception) # for logging
```

Comment:
```suggestion
        logger.info { exception }
```

slightly more performant